### PR TITLE
feat: page management and block ui redesign

### PR DESCRIPTION
**Pull request by lichunn**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

## Summary by CodeRabbit

- **New Features**
  - Enhanced user interface for lifecycle tips with a more integrated display.
  - Introduced a `tiny-popover` for improved interaction with block item settings.
  - Added a documentation link button for easier access to user guides.
  - Implemented collapsible sections in forms for better organization and usability.
  - Added SVG icons to buttons for improved visual representation.

- **Improvements**
  - Streamlined layouts and spacing adjustments for a more user-friendly design across multiple components.
  - Updated labels and placeholders for better clarity and user understanding.
  - Restructured forms with collapsible sections for better organization.

- **Styling Changes**
  - Modifications to CSS properties for buttons, headers, and various elements to enhance visual appeal and consistency.
  - Expanded color palette with new CSS variables for improved theming options.

### CODE REVIEW

File: packages/common/component/LifeCycles.vue
Comment (hexqi) [inline packages/common/component/LifeCycles.vue:1]: Huge PR！后面PR可以提小一点，大的PR太难检视了。

File: packages/common/component/MetaCodeEditor.vue [hunk 1]
```diff
@@ -120,6 +121,14 @@ export default {
       // 代码编辑器上方提示：title显示简短的文字描述，demo为显示的示例，点击 “展开示例” 可查看
       type: Object,
       default: () => ({ title: '', demo: '' })
+    },
+    shouBtnIcon: {
```
Comment (hexqi) [inline packages/common/component/MetaCodeEditor.vue]: 拼写错误，`shouBtnIcon` --> `showBtnIcon`

File: packages/common/component/MetaCodeEditor.vue [hunk 2]
```diff
@@ -120,6 +121,14 @@ export default {
...
+      type: Boolean,
+      default: false
+    },
+    svgName: {
+      type: String,
+      default: 'to-edit'
```
Comment (hexqi) [inline packages/common/component/MetaCodeEditor.vue]: 建议去掉showBtnIcon与svgName属性，改为一个插槽，CodeEditor是一个基础组件，最好不依赖svg图标

File: packages/plugins/bridge/src/Main.vue [hunk 3]
```diff
@@ -1,27 +1,18 @@
...
     <template #content>
-      <tiny-tabs v-model="activedName" class="tabs full-width-tabs" tab-style="button-card" @click="switchTab">
-        <tiny-tab-item :name="RESOURCE_TYPE.Util" title="工具类">
-          <bridge-manage ref="utilsRef" :name="RESOURCE_TYPE.Util" @open="openBridgePanel"></bridge-manage>
-        </tiny-tab-item>
-        <tiny-tab-item v-if="isVsCodeEnv" :name="RESOURCE_TYPE.Bridge" title="桥接源">
-          <bridge-manage ref="bridge" :name="RESOURCE_TYPE.Bridge" @open="openBridgePanel"></bridge-manage>
-        </tiny-tab-item>
-      </tiny-tabs>
+      <bridge-manage ref="utilsRef" :name="RESOURCE_TYPE.Util" @open="openBridgePanel"></bridge-manage>
```
Comment (hexqi) [inline packages/plugins/bridge/src/Main.vue:7]: 桥接源的UI既然删除了，那相应的js也删除了吧

File: packages/common/component/LifeCycles.vue
Comment (lichunn) [reply packages/common/component/LifeCycles.vue:1]: 好嘟

File: packages/plugins/datasource/src/DataSourceFieldCheckRanger.vue [hunk 4]
```diff
@@ -1,36 +1,43 @@
...
+        </div>
+      </tiny-col>
+    </tiny-row>
   </div>
 </template>
 
 <script>
 import { inject } from 'vue'
-import { Numeric, FormItem } from '@opentiny/vue'
+import { Numeric, FormItem, Row as TinyRow, Col as TinyCol } from '@opentiny/vue'
```
Comment (hexqi) [inline packages/plugins/datasource/src/DataSourceFieldCheckRanger.vue]: 风格保持统一，引入要么都用as，要么都不用

File: packages/plugins/datasource/src/DataSourceFieldForm.vue [hunk 5]
```diff
@@ -98,7 +99,11 @@ export default {
     const close = () => {
       state.isOpen = false
     }
-
+    const setId = () => {
+      if (props.isRow && !props.editable) {
+        return 'default-item'
```
Comment (hexqi) [inline packages/plugins/datasource/src/DataSourceFieldForm.vue]: setId这个名称不太合适，建议改为computed变量，然后使用名词命名变量

File: packages/plugins/datasource/src/DataSourceRecordList.vue [hunk 6]
```diff
@@ -95,8 +80,8 @@
 <script lang="jsx">
 import { reactive, ref, watchEffect, watch, computed } from 'vue'
 import { camelize, capitalize } from '@vue/shared'
-import { Grid, Pager, Input, Numeric, DatePicker, Switch, Slider, Link } from '@opentiny/vue'
-import { IconPlusCircle, IconImport } from '@opentiny/vue-icon'
+import { Grid, Pager, Input, Numeric, DatePicker, Switch, Slider, Link, Button } from '@opentiny/vue'
+import { IconPlusCircle, IconImport, iconPlus, iconUpload } from '@opentiny/vue-icon'
```
Comment (hexqi) [inline packages/plugins/datasource/src/DataSourceRecordList.vue]: IconPlusCircle, IconImport 这两个图标看起来没有用到，如果没用到，可以删掉

File: packages/plugins/i18n/src/Main.vue [hunk 7]
```diff
@@ -430,80 +433,83 @@ export default {
...
-          }
-        }
-      }
-    }
-
-    .tiny-grid__empty-text {
-      color: var(--ti-lowcode-toolbar-breadcrumb-color);
-    }
-  }
+  // :deep(.tiny-grid) {
```
Comment (hexqi) [inline packages/plugins/i18n/src/Main.vue]: 没用到代码可以删除

**Review by hexqi — APPROVED** [review]

**Pull request merged**